gem/lib/pupperware/spec_helper.rb

```diff
@@ -7,6 +7,8 @@
 module Pupperware
 module SpecHelpers
 
+  IS_WINDOWS = !File::ALT_SEPARATOR.nil?
```

Comment:
This isn't quite the same thing as `!!File::ALT_SEPARATOR`. The prefered check is `!!`

```
irb(main):005:0> !!false
=> false
irb(main):006:0> !false.nil?
=> true
```

Reply:
Ah yeah didn't notice they changed; I think "mine" is an earlier version of this check that I copied.

I'll update to `!!File::ALT_SEPARATOR`